src/test/java/uk/gov/hmcts/reform/sscs/factory/PersonalisationFactoryTest.java

```diff
@@ -38,7 +38,7 @@ public void setup() {
 
     @Test
     public void createPersonalisationWhenNotificationApplied() {
-        Personalisation result = factory.apply(APPEAL_RECEIVED_NOTIFICATION);
+        Personalisation result = factory.apply(ADJOURNED_NOTIFICATION);
```

Comment:
The test name could do with updating to what it's testing

Reply:
We have other events as well which could result base class Personalisation